support/cas-server-support-logging-config-cloudwatch/src/test/groovy/org/apereo/cas/logging/CloudWatchAppenderTests.groovy

```diff
@@ -0,0 +1,84 @@
+package org.apereo.cas.logging
```

Comment:
Just wondering, would Spock work here

Reply:
It would if there were no objections to adding another dependency. junit seems to work okay, though

Reply:
There are no objections from me

Reply:
This looks pretty good. Thanks Jj. 

Please do not use Spock and also, please rewrite the tests in Java.